### DISCUSSION TITLE
encrypted password in the database

### DISCUSSION THREAD

**Discussion by SteBakhita** [blueseerERP/blueseer#114] [Q&A] [answered]
Hi ,
Please, is there a way to encrypt users passwords in a database? Currently they are readable in the database. Thanks

**Comment by vaughnte** [comment]
The 'system' user passwords are currently encrypted and stored in the appropriate database table.   Please send an email to support@blueseer.com if you see where this is not the case.

**Comment by vaughnte** [comment, marked as answer]
...actually...you may be looking at the 'admin' password with the initial install.   the password is well documented for the admin so it remains 'admin' in the user password field....until you change it.   It then is encrypted before placing back in the database field.   Any new user added will have the password encrypted as well.   If you have any further questions...feel free to reach out to support@blueseer.com.

**Reply by SteBakhita** [reply]
Thank you!!
